common/src/main/java/org/corfudb/common/metrics/micrometer/registries/dropwizard/DropwizardDistributionSummaryCustom.java

```diff
@@ -0,0 +1,58 @@
+package org.corfudb.common.metrics.micrometer.registries.dropwizard;
+
+import io.micrometer.core.instrument.AbstractDistributionSummary;
+import io.micrometer.core.instrument.Clock;
+import io.micrometer.core.instrument.distribution.DistributionStatisticConfig;
+import io.micrometer.core.instrument.distribution.TimeWindowMax;
+import io.micrometer.core.instrument.util.MeterEquivalence;
+import io.micrometer.core.lang.Nullable;
+
+import java.util.concurrent.atomic.DoubleAdder;
+
+public class DropwizardDistributionSummaryCustom extends AbstractDistributionSummary {
```

Comment:
Would be great to have a comment for this class

Reply:
Renamed and added comments